Tasks/GulpV1/task.loc.json

```diff
@@ -0,0 +1,174 @@
+{
+  "id": "B82CFBE4-34F9-40F5-889E-C8842CA9DD9D",
+  "name": "Gulp",
+  "friendlyName": "ms-resource:loc.friendlyName",
+  "description": "ms-resource:loc.description",
+  "helpMarkDown": "ms-resource:loc.helpMarkDown",
+  "category": "Build",
+  "visibility": [
+    "Build"
+  ],
+  "runsOn": [
+    "Agent",
+    "DeploymentGroup"
+  ],
+  "author": "Microsoft Corporation",
+  "version": {
+    "Major": 0,
```

Comment:
Update to 1.0.0

Reply:
Oops. That was kind of the whole point, wasn't it.